.docker/scripts/init-ext.sh

```diff
@@ -37,4 +44,7 @@ installed_name=$(grep '^\s*name=' setup.py |sed "s|[^']*'\([-a-zA-Z0-9]*\)'.*|\1
 # Validate that the extension was installed correctly.
 if ! pip list | grep "$installed_name" > /dev/null; then echo "Unable to find the extension in the list"; exit 1; fi
 
+if [ -d "$SRC_DIR/ckan/ckanext/activity" ]; then
```

Comment:
is this ckan 2.10 item?

Reply:
Yes, the activity stream is being split from core and put into a plugin (but a plugin that is available out of the box, just needs to be enabled).